benchmark_np.py

```diff
@@ -74,28 +79,24 @@ def __len__(self):
         print("Testing: %s" % lib)
         for root, dirs, fnames in sorted(os.walk('AUDIO')):
             for audio_dir in dirs:
-                try:
-                    duration = int(audio_dir)
-                    dataset = AudioFolder(
-                        os.path.join(root, audio_dir),
-                        lib='load_' + lib,
-                        extension=args.ext
+                duration = int(audio_dir)
+                dataset = AudioFolder(
+                    os.path.join(root, audio_dir),
+                    extension=args.ext
+                )
+
+                # for fp in dataset.audio_files:
+                for fp in dataset.audio_files:
+                    time = timeit.timeit(
+                        functools.partial(test_np_loading, fp, lib),
+                        number=10
```

Comment:
Now you're iterating over the files, loading each one 10 times in a row, then storing the time it took to load the last file 10 times. I think you'd want to divide the return value by 10, and accumulate the time over all files? Also you may want to use `timeit.repeat()` to run 3 repetitions and then keep the smallest one, if the idea was to factor out disk I/O.

Reply:
Thanks Jan.  Will have some time this weekend to finish this up